src/main/webapp/i18n/en/exam.json

```diff
@@ -37,7 +37,7 @@
             "endExam": "Finish",
             "date": "Date",
             "time": "Time",
-            "durationWithFormat": "Duration (mm:ss)",
+            "durationWithFormat": "Duration",
```

Comment:
Same applies in the english translation https://github.com/ls1intum/Artemis/pull/3710/files#r667333310